### PR TITLE
Enzyme test & fix minor issues

### PR DESCRIPTION
**Pull request by ryanminjaeshin**
1. Enzyme test
2. Reflect the previous comment

### CODE REVIEW

**Comment by ryanminjaeshin** [comment]
Hi Team, 

I have tested the enzyme! 

A review will be appreciated :)

File: test/App.spec.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+describe('App', () => {
+  test('snapshot renders', () => {
+    const component = renderer.create(<App />);
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  // enzyme testing
+  it('renders the inner Counter', () => {
+    const wrapper = mount(<App />);
```
Comment (karebae) [inline test/App.spec.js:19]: this is interesting! guess this one in particular will be easy to replicate for our own components

File: test/App.spec.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    expect(wrapper.find(Counter).length).toEqual(1);
+  });
+});
+
+describe('Counter', () => {
+  test('snapshot renders', () => {
+    const component = renderer.create(<Counter counter={1} />);
+    const tree = component.toJSON();
```
Comment (karebae) [inline test/App.spec.js:27]: guess this means the react tree? so interesting

**Review by karebae — APPROVED** [review]
good jobbo ryan! looks good, excited to see what your actual tests will look like :D keep at it!

**Pull request merged**